specification.markdown

```diff
@@ -223,6 +223,7 @@ Within an EBML Schema the `<element>` uses the following attributes to define an
 | attribute name | required | definition |
 |:---------------|:---------|:-----------|
 | name           | Yes      | The official human-readable name of the EBML Element. The value of the name MUST be in the form of an NCName as defined by the [XML Schema specification](http://www.w3.org/TR/1999/REC-xml-names-19990114/#ns-decl). |
+| parent         | Yes      | The name of the parent EBML Elements. This value MUST be defined with the full hierarchy of EBML Elements separated with a dot. The top EBML Element in the hierarchy being the first in the value. Elements at the top level MUST use the 'none' parent value. Elements that can be found anywhere in the EBML Body or Header MUST use the 'any' parent value. |
```

Comment:
I like the general idea. I slightly dislike assigning special meaning to ordinary words (`none` and `any`). This prevents any other EBML-based specification from using those words as element names.

What about using symbols? E.g. `parent="-"` for no parent and `parent="*"` for it being allowed in any parent. Alternatively just leave `parent` empty for no parent.

Additionally I'd like to use an established method of specifying a path to an element, e.g. how [XPath's Location Path Expressions](http://www.w3schools.com/xsl/xpath_axes.asp). It's well-known and understood. To use your example from the pull request: `/Chapters/EditionEntry/ChapterAtom`